app/models/sign.rb

```diff
@@ -38,6 +38,10 @@ def playable_slides
     slides.shown.active.approved.ordered
   end
 
+  def unexpired_slides
+    sign_slides.unexpired
```

Comment:
Does anyone have an explanation for why this is a `has_many sign_slides` association *and* a `has_many slides` association? There is a destroy dependency with `slides`. But is there a reason for both cases? This seems like an anti-pattern to me. But I'm not sure.